tests/issues/issue-000-invalid-versions.php

```diff
@@ -8,12 +8,39 @@
         'properties' => [
             'Comment' => 'Firefox Generic',
             'Browser' => 'Firefox',
+            'Browser_Type' => 'Browser',
```

Comment:
I don't think we should add all of these properties to the tests in this file.

Reply:
reverted